actions-server/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:12.16.3-alpine
+EXPOSE 3000
+WORKDIR /usr/app
+COPY package.json .
+COPY yarn.lock .
+RUN yarn install --frozen-lockfile
```

Comment:
Dependencies are installed as part of building the image, and `node_modules` will solely live _inside_ of the container (backed by an anonymous Docker volume) which will be the main difference. This is good in that it will ensure Node.js versions are locked down and whatnot, but it also means that any package updates will need to happen _inside_ of the container (i.e. `docker exec actions-server`).